tee-worker/omni-executor/executor-core/src/native_task.rs

```diff
@@ -41,7 +39,7 @@ pub type PumpxChainId = u32;
 #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
 pub enum NativeTask {
 	RequestAuthToken(Identity),
-	RequestIntent(Identity, IntentId, Intent),
+	RequestIntent(AccountId, IntentId, Intent),
 	CreateAccountStore(Identity),
```

Comment:
Is it possible that remaining task types will accept `AccountId` instead of `Identity` ?

Reply:
Yes, ok